src/components/tools/hooks/use-measure-text.ts

```diff
@@ -1,15 +1,20 @@
-import { useCallback, useRef } from "react";
+import { useCallback } from "react";
 
-export const useMeasureText = (font: string) => {
-  const cache = useRef<Record<string, number>>({});
-  const canvas = useRef(document.createElement("canvas"));
-  const context = canvas.current.getContext("2d");
+const defaultFont = "italic 14px Lato, sans-serif";
+const canvas = document.createElement("canvas");
+const cache: Record<string, Record<string, number>> = {};
+
+export const measureText =
+  (text:string, font = defaultFont) => {
```

Comment:
```suggestion
export const measureText = (text: string, font = defaultFont) => {
```
Don't need to break the line any more